src/components/GroupSystems/GroupSystems.js

```diff
@@ -126,6 +126,8 @@ const GroupSystems = ({ groupName, groupId }) => {
           }}
           groupId={groupId}
           groupName={groupName}
+          edgeParityIsAllowed={true}
+          initialActiveTab={0}
```

Comment:
I think we need to give a proper tab key instead of numbers considering initialActiveTab is passed from another component and it is not easily known what tab is active exactly.